couch/check.py

```diff
@@ -203,27 +278,39 @@ def _get_instance_names(self, server, instance):
         else:
             return [name]
 
+    def _get_dbs_to_scan(self, server, instance, name, tags):
+        dbs = self.agent_check.get(urljoin(server, "_all_dbs"), instance, tags)
+        nodes = self.agent_check.get(urljoin(server, "_membership"), instance, tags)['cluster_nodes']
+        idx = nodes.index(name)
+        size = int(math.ceil(len(dbs) / float(len(nodes))))
+        return dbs[(idx * size):((idx + 1) * size)]
+
     def check(self, instance):
         server = self.agent_check.get_server(instance)
 
         max_dbs_per_check = instance.get('max_dbs_per_check', self.agent_check.MAX_DB)
         for name in self._get_instance_names(server, instance):
             tags = ["instance:{0}".format(name)]
             self._build_metrics(self._get_node_stats(server, name, instance, tags), tags)
+            self._build_system_metrics(self._get_system_stats(server, name, instance, tags), tags)
+            self._build_active_tasks_metrics(self._get_active_tasks(server, name, instance, tags), tags)
 
             db_whitelist = instance.get('db_whitelist', None)
             db_blacklist = instance.get('db_blacklist', [])
             scanned_dbs = 0
-            for db in self.agent_check.get(urljoin(server, "/_all_dbs"), instance, tags):
+            for db in self._get_dbs_to_scan(server, instance, name, tags):
                 if (db_whitelist is None or db in db_whitelist) and (db not in db_blacklist):
-                    tags = ["instance:{0}".format(name), "db:{0}".format(db)]
```

Comment:
Why are you removing this tag? I don't want this removal to break monitors for anyone.